components/camel-atmosphere-websocket/src/test/java/org/apache/camel/component/atmosphere/websocket/WebsocketRouteWithInitParamTest.java

```diff
@@ -110,18 +114,18 @@ void testWebsocketSingleClientBroadcastMultipleClientsGuaranteeDelivery() throws
         final int awaitTime = 5;
         connectionKeyUserMap.clear();
 
-        WebsocketTestClient wsclient1 = new WebsocketTestClient("ws://localhost:" + PORT + "/hola3", 2);
-        WebsocketTestClient wsclient2 = new WebsocketTestClient("ws://localhost:" + PORT + "/hola3", 2);
-        WebsocketTestClient wsclient3 = new WebsocketTestClient("ws://localhost:" + PORT + "/hola3", 2);
+        WebsocketTestClient wsclient1 = new WebsocketTestClient("ws://localhost:" + PORT + "/guarantee", 2);
+        WebsocketTestClient wsclient2 = new WebsocketTestClient("ws://localhost:" + PORT + "/guarantee", 2);
+        WebsocketTestClient wsclient3 = new WebsocketTestClient("ws://localhost:" + PORT + "/guarantee", 2);
 
         wsclient1.connect();
-        wsclient1.await(awaitTime);
+        wsclient1.await(1);
```

Comment:
in this way the test runs faster, I noticed that when connecting a high await time is not useful, moreover, there are 3 consequent await

Reply:
Hm, I am afraid this may cause problems in our Apache CI. Our nodes there are not so fast, so the the time needed to receive the messages could be higher.